config_quick_debug.yml

```diff
@@ -3,7 +3,6 @@ n_repeats: 1
 scenario_params_list:
  - dataset_name:
     - 'mnist'
```

Comment:
I would leave at least 2 datasets for this config, so at least 2 datasets will be run by Travis.

Reply:
Done, cifar10 is back for travis